src/NUnitFramework/framework/Internal/Execution/SimpleWorkItemDispatcher.cs

```diff
@@ -60,7 +60,16 @@ public void Start(WorkItem topLevelWorkItem)
 
 #if APARTMENT_STATE
             if (topLevelWorkItem.TargetApartment == ApartmentState.STA)
+            {
+#if NETSTANDARD2_0
+                if (!OSPlatform.CurrentPlatform.IsWindows)
+                {
+                    topLevelWorkItem.MarkNotRunnable("Apartment state cannot be set on non-windows platforms.");
```

Comment:
‘Windows’ is a brand name, so we should follow the brand's capitalization. 

How about `on platforms other than Windows.`?

(Two places.)